torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -357,37 +400,40 @@ void ExportModule(
     const Module& module,
     std::ostream& out,
     const ExtraFilesMap& extra_files,
-    bool bytecode_format) {
+    bool bytecode_format,
+    bool save_debug_info) {
   ScriptModuleSerializer serializer(
       [&](const void* buf, size_t nbytes) -> size_t {
         out.write(static_cast<const char*>(buf), nbytes);
         return !out ? 0 : nbytes;
       });
-  serializer.serialize(module, extra_files, bytecode_format);
+  serializer.serialize(module, extra_files, bytecode_format, save_debug_info);
 }
 
 void ExportModule(
     const Module& module,
     const std::string& filename,
     const ExtraFilesMap& extra_files,
-    bool bytecode_format) {
+    bool bytecode_format,
+    bool save_debug_info) {
   ScriptModuleSerializer serializer(filename);
-  serializer.serialize(module, extra_files, bytecode_format);
+  serializer.serialize(module, extra_files, bytecode_format, save_debug_info);
 }
 
 void ExportModule(
     const Module& module,
     const std::function<size_t(const void*, size_t)>& writer_func,
     const ExtraFilesMap& extra_files,
-    bool bytecode_format) {
+    bool bytecode_format,
+    bool save_debug_info) {
   ScriptModuleSerializer serializer(writer_func);
-  serializer.serialize(module, extra_files, bytecode_format);
+  serializer.serialize(module, extra_files, bytecode_format, save_debug_info);
 }
 
 namespace {
 void export_opnames(const script::Module& m, std::set<std::string>& opnames) {
   std::vector<c10::IValue> elements;
-  moduleMethodsTuple(m, elements);
+  moduleMethodsTuple(m, elements, false /* save_debug_info */);
```

Comment:
Good putting a comment there.

Reply:
Just a quick question. Here I assumed that we didn't need to save debug info when calling `export_opnames`. Was that correct?

Reply:
I think so. @iseeyuan?